assets/js/translator.js

```diff
@@ -132,6 +133,12 @@ if(modeEnabled('translation')) {
             }, timeout);
 
             synchronizeTextareaHeights();
+
+            if(isURL($('#originalText').val())) {
+                setTimeout(function () {
+                    translateWebpage();
```

Comment:
I don't really mind if the behavior is consistent. i.e. no switch to website translation without instant translation being checked.